taipy/gui_core/_context.py

```diff
@@ -860,6 +866,7 @@ def act_on_jobs(self, state: State, id: str, payload: t.Dict[str, str]):
             _GuiCoreContext.__assign_var(state, payload.get("error_id"), "<br/>".join(errs) if errs else "")
 
     def get_job_details(self, job_id: t.Optional[JobId]):
+        job = None
```

Comment:
Why?

Reply:
because ruff/mypy/whoever complained that in the exception block job could be unbound

Reply:
not necessary with your proposed change